### ISSUE TITLE
Error: control/filtering/set_rules | Failed to read request body

### ISSUE THREAD

**Issue by tom-b1995** [AdguardTeam/AdGuardHome#2675]
### Issue Details

Hello everyone,

I have installed AdGuard Home on 21 August 2020 on my router Turris Omnia. I have been using it  without any issues whatsoever since then. The filtering or blocking worked (and still works) as expected. So, I created my own set of rules in a tab _Filters_ - _Custom filtering rules_ which worked great until yesterday (12.02.2021). When I tried to add an exception and unblock a domain, I received an error message: 
**Error: control/filtering/set_rules | Failed to read request body: attempted to read more than 65536 bytes | 400**

The error prevents me from editing my _Custom filtering rules_. I cannot add, remove or change anything. When I click on the _Apply_ button after making some change, it returns an error message described as above. The blocking or allowing a certain domain (from my _Custom filtering rules_) is, however, still fully functional.

The error message appeared right after the new update (v0.105.0). There was no such error before this update. I realize my file could be quite large (907 KB in .txt) and has approximately more than 41 500 lines, but everything worked before the update. The router does not have full storage, it has more than 3 GB of space remaining. The router itself is not overloaded and does not report any issues. I edit the list only via Copy and Paste from a plain .txt file stored in my computer using web interface of AdGuard Home from web browser. Rebooting does not help.

* **Version of AdGuard Home server:**
  * Version: v0.105.0
* **How did you install AdGuard Home:**
  * I have followed a guide on OpenWrt forum
* **How did you setup DNS configuration:**
  * I have installed AdGuard Home and configured it as a DNS server on my main router, Turris Omnia.
* **If it's a router or IoT, please write device model:**
  * Turris Omnia 1 GB No Wi-Fi - The full documentation of the device is here: https://wiki.turris.cz/doc/_media/omnia/omnia_datasheet_1gb_rgb.pdf
* **CPU architecture:**
  * Marvell Armada 385, dual-core 1.6 GHz, ARMv7
* **Operating system and version:**
  * Firmware Version: OpenWrt omnia 15.05 r47055 / LuCI fd63d5cc7dfc69c640b741dceb80555141a80670 branch (git-19.318.61483-fd63d5c)
  Kernel Version: 4.4.199-a890a5a94ebb621f8f1720c24d12fef1-1

### Expected Behavior
I would like to know if this issue could be fixed - So I could edit my custom filters again, if possible.

### Actual Behavior
The current error prevents me from editing my _Custom filtering rules_. I cannot remove a line with a domain or add another one (for both rules - block and unblock). However, the _Custom filtering rules_ are still working - they still allow or block particular domain according to their rules.

### Screenshots
![Screenshot 2021-02-13 023502](https://user-images.githubusercontent.com/78988374/107838561-b2ff3300-6da6-11eb-9b55-912504ef36eb.png)
![Screenshot 2021-02-13 023625](https://user-images.githubusercontent.com/78988374/107838565-b85c7d80-6da6-11eb-83c1-09e9391044df.png)

### Additional Information
Unfortunately, I am not a programmer or an IT specialist so I cannot help you much more, without additional help from you. I would be very grateful if someone would have read this message and will try to help me or fix the issue in the next build. Thank you very much for your time and attention.

**Comment by HorayNarea** [comment]
The same problem exists for the "disallowed clients" list… I think the allowed http body size should not be so low if you are logged in

**Comment by ainar-g** [comment]
Should be fixed as of snapshot aebcd74e.  Can you please check if our solution fixes the issue for you?

**Comment by tom-b1995** [comment]
> Should be fixed as of snapshot [aebcd74](https://github.com/AdguardTeam/AdGuardHome/commit/aebcd74efe5fedfcf4db7229d171471c26dd8a69). Can you please check if our solution fixes the issue for you?

Good morning, Mr. Garipov,

Sorry for writing so late, but I **confirm** the problem was **fix**ed. I updated my AdGuard Home to version v0.105.1 and everything is fully functional again. The list of custom filters can be edited again, no problems so far.
![image](https://user-images.githubusercontent.com/78988374/108002286-74a88480-6fef-11eb-96cb-2b72e36e3d29.png)
![image](https://user-images.githubusercontent.com/78988374/108002308-838f3700-6fef-11eb-9a8f-24a9e354dc9b.png)

Thank You and all the developers so much for the fix and your time. I am sorry I could not help you more. I am beginner in programming, so I tried to report this issue at least. 
Anyways, have a nice day and wish you good luck with the development of AdGuard!
Best regards, 
Tom B.
